db/migrate/20181109120333_create_school_groups_and_scoreboards.rb

```diff
@@ -0,0 +1,22 @@
+class CreateSchoolGroupsAndScoreboards < ActiveRecord::Migration[5.2]
+  def change
+
+    create_table :scoreboards do |t|
+      t.string :name, null: false
```

Comment:
@jamesjefferies as far as I can see, this gets translated into an unrestricted `varchar`/`character varying`, which behaves the same as a a `text` type. I can change them to `text` if you think that makes it clearer?

Reply:
@jamesalmond nope - if string has unrestricted length then I'm happy for it to be string and I'll make a mental note that we can use string instead of text if we like